### ISSUE TITLE
bug: Windows 10 action failed

### ISSUE THREAD

**Issue by hydai** [WasmEdge/WasmEdge#1290]
## Description

The building CI is failed from last week. Not sure which part of the environment is changed.

```bash
ninja: error: 'C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/DIA SDK/lib/amd64/diaguids.lib', needed by 'test/aot/wasmedgeAOTCoreTests.exe', missing and no known rule to make it
```

### Expected

The building process should be fine.

**Comment by LFsWang** [comment]
I am not sure why there is a hardcode path. replace it with local version will fix the build.

https://github.com/WasmEdge/llvm-windows release
```LLVM-13.0.0-win64\lib\cmake\llvm\LLVMExports.cmake``` L385

It is fixed for me.
 ```C:/Program Files/Microsoft Visual Studio/2022/Community/DIA SDK/lib/amd64/diaguids.lib```

**Comment by hydai** [comment]
Fixed in #1330

**Comment by abhinandanudupa** [comment]
I think this needs to be reopened as I am getting the same error on the latest build (and the solution by @LFsWang works).
